src/models/oryp9/repairs.md

```diff
@@ -0,0 +1,252 @@
+
+- Thermal paste should generally be replaced after removing the heatsink; thermal paste also dries over time, so depending on your climate and the age of the machine, applying new thermal paste may increase thermal performance.
+- The heat-conductive metal does not dry out, and replacing it is optional; replacement may be useful if the previous metal has flowed out of the proper area for cooling.
+
+Replacement heat-conductive metal fitted to the machine and replacement fans can be obtained by [contacting support](https://support.system76.com).
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 20 minutes  
+**Difficulty:** High <span style="color:red;">●</span>  
+
+### Steps to replace the fans/heatsink/thermal paste:
+
+1. Power off the machine. Wait 20 minutes for the machine to cool down to room temperature.
+    - **Warning:** Removing the heatsink before the machine has cooled to room temperature may result in heat-conductive metal flowing onto the motherboard and causing irrepairable damage.
+2. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover).
+3. Unscrew the fan and heatsink screws in the order of the stamped numbers, starting with #1, then #2, and continuing until you have removed #12.
+    - The screws are held captive and will not come out of the fans/heatsink. The fan screws will click when they are fully loosened.
+    - The order for the 17" model is shown below; the screws are numbered differently on the 15" model. Follow the order stamped into the heatsink metal.
+    - Do not remove the smaller screws holding the fan covers onto the fans.
+
+![Thermal screws](./img/thermal-screws.webp)
+
+4. If there is any clear tape securing the heatsink/fans or the fan wires, remove it.
+5. Unplug the two white fan connectors from the motherboard (highlighted red above).
+6. Remove the heatsink/fans from the case, being careful not to bend the heatsink pipes. It may take some pressure to break the seal of the thermal paste.
+7. Using a paper towel, remove the existing thermal paste from the integrated CPU die, the border of the CPU chip, the CPU heatsink, the border of the GPU die, and the GPU heatsink.
+    - A small amount of rubbing alcohol can be used if the old paste is dried or difficult to remove.
+    - The heat-conductive metal may have some holes where pieces stuck to the die or heatsink.
+    - If you are not replacing the heat-conductive metal, do not pick or wipe the heat-conductive metal off of the main CPU die or the GPU heatsink.
+
+![Thermal paste removal](./img/thermal-paste-removal.webp)
+
+8. If you are replacing the heat-conductive metal, remove the plastic covers from the GPU heatsink and the CPU chip.
+    - A fingernail or a thin plastic tool can be used to pick a side of the plastic cover up from the CPU chip/GPU heatsink.
```

Comment:
There should be a warning about not getting the metal stuff anywhere on the motherboard, but I'm not sure whether it works better here or closer to the top, near the room temperature warning.

Reply:
I'll copy some of the warning under Step 1 down here as well.

Reply:
Let me know how it looks with [a7ca2dd](https://github.com/system76/tech-docs/pull/140/commits/a7ca2dd58fb26ddd7e6b11427ea0429ccc6fb814). I almost put "if powered on at room temperature," but I've personally killed a board without powering it on (and the thermal instructions don't include unplugging the battery), so this seems safer.

Reply:
Looks good to me!